lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -48,11 +51,14 @@ class LoadFastEnough4Pwa extends Audit {
     // If throttling was default devtools or lantern slow 4G throttling, then reuse the given settings
     // Otherwise, we'll force the usage of lantern slow 4G.
     const settingOverrides = {throttlingMethod: 'simulate', throttling: mobileThrottling};
-    const settings =
-      context.settings.throttlingMethod !== 'provided' &&
-      isDeepEqual(context.settings.throttling, mobileThrottling)
-        ? context.settings
-        : Object.assign({}, context.settings, settingOverrides);
+
+    // Override settings for interactive if provided throttling was used and network
+    // throttling was not consistent with standard `mobile network throttling`
+    const override = context.settings.throttlingMethod === 'provided' &&
```

Comment:
nit: mind naming this `shouldOverride` or something a bit more clear it's a boolean rather than the override itself :)

Reply:
is the idea you can be `provided`, but if you *say* you were slow enough in `settings.throttling` you'll pass?

Reply:
oh whoops thanks @brendankenny I didn't notice the conditions got only half flipped in this

@exterkamp  you meant to `!` this whole condition right? we need to flip the `&&` to `||`

the original idea was that you are fine if you are..
A) Using default simulated settings
OR 
B) Using default devtools settings

i.e. if you are using provided OR you don't have the mobileThrottling settings, you need to be overriden

Reply:
Done and done!  I misread the original if statement.